.github/workflows/linting.yml

```diff
@@ -63,16 +63,34 @@ jobs:
   markdown-link-check:
     name: Markdown Links (modified files)
     runs-on: ubuntu-latest
+    env:
+      BASE_BRANCH: ${{ github.base_ref }}
+      FILE_EXTENSION: .md
     steps:
       - name: Check out the repository
         uses: actions/checkout@v2
-
+      - name: Extract markdown from CSV.yaml
+        run: |-
+          sudo snap install yq --channel=v4/stable && \
+          yq e '.spec.description, .spec.links' config/manifests/bases/submariner.clusterserviceversion.yaml > bundle.md
```

Comment:
Maybe we should only create/check the YAML links if that file is modified? Maybe we should always check it in the periodic link check test?

Reply:
I think the second option covers more.